modules/tinymce/src/core/main/ts/selection/ElementSelection.ts

```diff
@@ -46,6 +46,10 @@ const skipEmptyTextNodes = (node: Node, forwards: boolean) => {
   return node || orig;
 };
 
+const isAnchorPrevSibling = (element: Node): boolean =>
+  Traverse.prevSibling(SugarElement.fromDom(element))
+    .exists((node) => SugarNode.name(node) === 'a');
```

Comment:
I believe you can use `SugarNode.isTag` here

```suggestion
Traverse.prevSibling(SugarElement.fromDom(element)).exists(SugarNode.isTag('a'));
```